src/sagas/networkSettings.js

```diff
@@ -289,6 +289,8 @@ export function* persistNetworkSettings(action) {
     return;
   }
 
+  // Remove registered tokens only
+  yield call([wallet.storage, wallet.storage.cleanStorage], false, false, true)
   // Stop wallet and clean its storage without clean its access data.
   wallet.stop({ cleanStorage: true, cleanAddresses: true });
```

Comment:
The method `stop` from the wallet already calls the `cleanStorage`. With this change we are calling it twice.

We should add a new parameter to the `stop` function to support `cleanTokens` as well.

Reply:
Oh, ok! I will open an issue for it.

Reply:
See https://github.com/HathorNetwork/internal-issues/issues/290

Reply:
I think we should do this in this PR, it's a problem to merge like this

Reply:
So, are we going to do this in this PR? Was it solved in a different way?

Reply:
Done